### ISSUE TITLE
Is extension going through Moz Recommended Extensions Program

### ISSUE THREAD

**Issue by PresGas** [keepassxreboot/keepassxc-browser#1036]
Is extension going through the Mozilla Recommended Extensions Program

https://support.mozilla.org/en-US/kb/recommended-extensions-program

## Expected Behavior
Extension should be available and installable on all supported versions of FF including Android

## Current Behavior
Extension is not considered "Recommended" and therefore not only gets the banner:
"This add-on is not actively monitored for security by Mozilla. Make sure you trust it before installing."
...for desktop, but also is not available at all in the recent GeckoView/Android version of Firefox.

**Comment by varjolintu** [comment]
I know few have recommended KeePassXC-Browser to that program, but I have no idea how the progress actually goes.

**Comment by PresGas** [comment]
I think you could start by reaching out to Caitlin Neiman - Add-ons Community Manager at Mozilla.
https://blog.mozilla.org/addons/author/cneimanmozilla-com/

Otherwise, I do feel like they aren't making it too easy to do so. If you are connected with some of the addon developers that DO have Recommended Extensions, you could see what they have to say.

Otherwise, you could do a general query to the Electronic Frontier Foundation. Both Privacy Badger and HTTPS Everywhere are Recommended Extensions and may know the process.

**Comment by droidmonkey** [comment]
We are recommended by the EFF

**Comment by PresGas** [comment]
@droidmonkey yes, you may be recommend by the EFF, but even their addons had to go through whatever Mozilla does to get added to the Recommended Extensions Program...as that is by Mozilla, not the EFF.

**Comment by hahjortland** [comment]
I guess the most important thing is the security review, so being a Verified Extension is perhaps almost as good as being a Recommended Extension.

From https://support.mozilla.org/en-US/kb/add-on-badges:
> Verified extensions are built by third-party developers and are reviewed for security and policy compliance. Developers of Verified extensions pay a fee to have their code reviewed.
> [...]
> Participation in this program is currently limited due to our capacity for code reviews, but we may expand in the future if more developers are interested in participating.

**Comment by PresGas** [comment]
Yes, it is agreed for security that Verified is good. However, if it is to be available on FF for Andriod, it needs Recommended according to Mozilla. This would then put it on par with things like Mozilla's lockwise et al.

**Comment by droidmonkey** [comment]
There is no point to this extension on android.

**Comment by PresGas** [comment]
Possible use case: "Unified" database using syncthing across desktop and android. 

What android app would be recommended then?

**Comment by PresGas** [comment]
Nevertheless for the reasons above, it would be valuable to have it go all the way to Recommended if it was on android or not:

https://github.com/keepassxreboot/keepassxc-browser/issues/1036#issuecomment-706607075

**Comment by AkechiShiro** [comment]
Is this issue still relevant today ? I don't see the banner anymore @PresGas

**Comment by PresGas** [comment]
Yes, if it has the banner warning about security and trust it is not Recommended either.

![image](https://github.com/keepassxreboot/keepassxc-browser/assets/4943849/227c2276-3060-45ba-8b5f-b2fcbe8f31ae)
